frontend/src/pages/Share/Share.tsx

```diff
@@ -35,7 +35,7 @@ const Share = () => {
         break
       }
       case 'module': {
-        decodeModule(data).then((decodedJson) => {
+        decodeData(data).then((decodedJson) => {
           const dataJson = new File([JSON.stringify(decodedJson)], 'Shared Project')
           useParseModuleFile(onModule, t('load_fail'), dataJson, handleLoadSuccess, handleLoadFail)
```

Comment:
These two functions look very similar

Might be worth chucking in backlog for future people to notice
https://github.com/automatarium/automatarium/blob/7a452c3a189a56ae30b4eec393638c7c549263a6/frontend/src/hooks/useActions.ts#L614-L642 
https://github.com/automatarium/automatarium/blob/7a452c3a189a56ae30b4eec393638c7c549263a6/frontend/src/hooks/useActions.ts#L697-L723